src/components/Loader.vue

```diff
@@ -0,0 +1,40 @@
+<!--
+ - @copyright Copyright (c) 2020 Nextcloud
+ -
+ - @license GNU AGPL version 3 or any later version
```

Comment:
Nope, you're the owner!

Reply:
Btw please also add yourself as the author for the other files you edited

Reply:
:disappointed:

Reply:
done